test/custom-parser.4.test.js

```diff
@@ -230,10 +231,10 @@ test('removeAllContentTypeParsers should support encapsulation', t => {
         'Content-Type': 'application/json'
       }
     }, (err, response, body) => {
-      t.error(err)
-      t.equal(response.statusCode, 200)
-      t.same(JSON.parse(body.toString()).test, 1)
-      fastify.close()
+      t.assert.ifError(err)
+      t.assert.strictEqual(response.statusCode, 200)
+      t.assert.equal(JSON.parse(body.toString()).test, 1)
+      testDone()
```

Comment:
We must use the `test/toolkit.js` here too

Reply:
Addressed in https://github.com/fastify/fastify/pull/6042/commits/2154944b54da40f4e9a3a94693f3db95e3504b37